lib/data/benefit_cap_data.yml

```diff
@@ -1,4 +1,21 @@
 ---
-single: 350
-couple: 500
-parent: 500
+rates:
+  single: 350
+  couple: 500
+  parent: 500
+benefits:
```

Comment:
I don't think it makes sense to add these benefits in this commit as they're not being used for anything.